assessments/projects/spiffe-spire/outline.md

```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
 
 ## Background
 
-SPIFFE and SPIRE are a set of APIs and associated tooling that provides a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
+Secure Protection Identity Framework for Everyone (SPIFFE) are a set of APIs with SPIRe as the production grade implemtnation and associated tooling that provide a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
```

Comment:
nit: 
SPIRE
implemtnation typo

"the production grade" - is the project is making this statement?

Reply:
corrected, changed to production-ready